php_fpm/check.py

```diff
@@ -44,6 +44,10 @@ def check(self, instance):
         tags = instance.get('tags', [])
         http_host = instance.get('http_host')
 
+        timeout = instance.get('timeout')
+        if timeout is None:
```

Comment:
Since it's possible to specify the default in the `get` function directly, could you do it there?
```python
timeout = instance.get('timeout', self.DEFAULT_TIMEOUT)
```